apps/docs/pages/guides/database/column-encryption.mdx

```diff
@@ -3,20 +3,17 @@ import Layout from '~/layouts/DefaultGuideLayout'
 export const meta = {
   id: 'column-encryption',
   title: 'Column Encryption',
-  description: 'Use Supabase to store and serve files.',
+  description: 'Encrypt columns using Transparent Column Encryption.',
+  subtitle: 'Encrypt columns using Transparent Column Encryption.',
   sidebar_label: 'Overview',
   video: 'https://www.youtube.com/v/J9mTPY8rIXE',
 }
 
-Encrypted Columns for Tables
+Supabase provides a secure method for encrypting columns using [Vault](/docs/guides/database/vault), our Postgres secrets manager. Vault is Postgres extension with an integrated UI intended to act as a secure global secrets management for you project.
 
-## Transparent Column Encryption (TCE)
+Vault enables an advance feature called Transparent Column Encryption (TCE) which provides a safe way to encrypt your data so that it doesn't leak into logs and backups. TCE provides a safe way to encrypt your data so that it doesn't leak into logs and backups. It can also provide row-level authenticated encryption.
```

Comment:
```suggestion
Vault enables an advanced feature called Transparent Column Encryption (TCE) which provides a safe way to encrypt your data so that it doesn't leak into logs and backups. It can also provide row-level authenticated encryption.
```